source/connect/mongoclient.txt

```diff
@@ -18,54 +18,70 @@ Create a MongoClient
 
-.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png
-   :alt: Each part of the connection string
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
 
-In this example, we connect to an Atlas MongoDB deployment that has a
-DNS SRV record. For more details, see the
-:manual:`DNS Seed List Connection Format
-</reference/connection-string/#dns-seed-list-connection-format>`
-documentation. This format offers flexibility in deployment and the
-ability to change the servers in rotation without reconfiguring clients.
+   * - Component
+     - Description
 
-.. note::
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
 
-   To learn how to retrieve your connection string in Atlas, see the
-   :atlas:`Atlas driver connection guide </driver-connection>`.
+   * - ``username:password``
 
-If you are connecting to an instance or replica set that does not have a
-DNS SRV address, you must use ``mongodb`` for the protocol, which specifies
-the :manual:`Standard Connection String Format
-</reference/connection-string/#std-label-connections-standard-connection-string-format>`.
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+       For more information about the ``authSource`` connection option, 
+       see :ref:`node-troubleshooting-connection-admin` in the Connection Troubleshooting guide.
 
-After the protocol, the next part of the connection string contains credentials
-if you are using password-based authentication. Replace the value of ``user``
-with your username and ``pass`` with your password. If you are using an
-authentication mechanism that does not require a username and password, omit
-this part of the connection URI.
+   * - ``host[:port]``
 
-The next part of the connection string specifies the hostname or IP address of
-your MongoDB instance, followed by the port number. In the example above, we use
-``sample.host`` as the hostname and ``27017`` as the port. Replace these values
-to point to your MongoDB instance.
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
 
-The last part of the connection string contains connection and authentication
-options as parameters. In the example above, we set two connection options:
-``maxPoolSize=20`` and ``w=majority``. For more information on connection
-options, skip to the :ref:`node-connection-options` section.
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
```

Comment:
Maybe worth mentioning this is the database that is used when calling `client.db()` with no argument?